samples/kubernetes/end2end/docs/03-wls-operator.md

```diff
@@ -83,11 +83,11 @@ kubectl get crd domains.weblogic.oracle
 NAME                      CREATED AT
 domains.weblogic.oracle   2019-05-28T07:17:26Z
 ```
-Verify the domain CRD version is correct.
+Verify that the domain CRD version is correct.
 ```
 kubectl get crd domains.weblogic.oracle -o jsonpath="{.spec.version}"
 ```
-The expected version should be `v5`.
+The expected version should be `v5`. Since the WebLogic operator is backward compatible, domain resource with version older than `v5` should be supported.
```

Comment:
Since the WebLogic operator is backward compatible, domain resource with version -> Because the WebLogic operator is backward compatible, a domain resource with any version

Reply:
Fixed.

Reply:
Not really. I wanted you to change the word "Since" to "Because." According to the Oracle style guide, "since – don't use to mean for the reason of; use because. Use since only for matters of time, for example, since the last time the program was run."  Also, you need to put an article "a" before the words domain resource, for example, "a domain resource."